### ISSUE TITLE
lr is always 0 in fully async

### ISSUE THREAD

**Issue by mikequan0425** [verl-project/verl#6683]
### System Info

main verl

### Information

- [ ] The official example scripts
- [x] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [x] My own task or dataset (give details below)

### Reproduction

Whether using FSDP or VeOmni, the training step is read from the optim config when building the learning rate scheduler.
| pic1 | pic2 |
|-----|-----|
| <img width="580" alt="Image" src="https://github.com/user-attachments/assets/1b471ef2-5058-4e52-bc2e-3c25f915f0c9" /> | <img width="580" alt="Image" src="https://github.com/user-attachments/assets/2d247381-9076-4205-a3f7-ff994c1d92b7" /> |

The running script is modified based on the sync mode. Tasks started via main_ppo do not have this issue. 
Code inspection reveals that this branch is executed when creating the trainer under fully async mode：
`verl.workers.engine.veomni.transformer_impl.VeOmniEngine._build_lr_scheduler`
where will do：`train_steps=optim_config.total_training_steps`，the default value of optim_config.total_training_steps here is -1.

Tasks started through main_ppo assign trainer.total_training_steps to optim_config.total_training_steps：
<img width="1280" height="712" alt="Image" src="https://github.com/user-attachments/assets/50cb433e-ff4b-4125-ae3d-87d674189d43" />

In Async mode, It can be observed that a sync about  total_train_steps between rollouter and trainer after create trainer. But yet this operation no longer takes effect on optim.lr at this point. If actor_rollout_ref.actor.optim.total_training_steps is not configured separately, the lr will remain 0 all the time.

<img width="1280" height="911" alt="Image" src="https://github.com/user-attachments/assets/a6fbdc16-c189-4e24-a4f8-354b238c56c3" />

### Expected behavior

The optim_config.total_training_steps is automatically defined by the rollout.total_rollout_steps.
